test/e2e/util/util.go

```diff
@@ -64,6 +64,26 @@ func GetTunedForNode(cs *framework.ClientSet, node *corev1.Node) (*corev1.Pod, e
 	return &podList.Items[0], nil
 }
 
+// GetTunedForNode returns a Pod that runs on a given node.
+func GetNodeTuningOperatorPod(cs *framework.ClientSet) (*corev1.Pod, error) {
+	listOptions := metav1.ListOptions{
+		LabelSelector: labels.SelectorFromSet(labels.Set{"name": "cluster-node-tuning-operator"}).String(),
+	}
+
+	podList, err := cs.Pods(ntoconfig.OperatorNamespace()).List(context.TODO(), listOptions)
+	if err != nil {
+		return nil, fmt.Errorf("couldn't list potential NTO operator pods: %v", err)
```

Comment:
I *think* it should be `s/pods/Pods/`.  At least that what I use in the logs elsewhere.  Similarly below.

Reply:
Fixed in a few spots, thanks.